src/services/repo/useRepo.tsx

```diff
@@ -121,14 +121,12 @@ export function useRepo({ provider, owner, repo, opts = {} }: UseRepoArgs) {
           }
         }
 
-        return (
-          {
```

Comment:
this was a random fix, right side of expression would never be hit since we're always returning an object